src/docs/release/breaking-changes/forgetchild-call-super

```diff
@@ -0,0 +1,84 @@
+---
+title: Element subclasses must call super in forgetChild
+description: Any element subclasses that override forgetChild will require to call super.
+---
+
+## Summary
+
+A recent global key duplication detection refactor requires `Element` subclasses which override
+`forgetChild` to call super.
+
+## Context
+
+When we encounter a global key duplication which will be cleaned up by an element rebuild later, we
+must not report global key duplication. Our current implementation throws error as soon as we detect
+such duplication and does not wait for the rebuild if the element with duplicated global key will
+rebuild. We would like to change it so that it keeps track all the global key duplications during a
+build and only verify global key duplication at the end of the build instead of throwing error
+immediately. In order to correctly differentiate between a real global key duplication and the case
+we mentioned earlier, we would implement a mechanism to remove previous global key duplication in
+forgetChild if the rebuild has happened. This will, however, require all Element subclasses that
+override forgetChild to call the super method.
+
+## Description of change
+
+The `forgetChild` of abstract class `Element` has a base implementation to remove global key reservation,
+and is enforced by ``@mustCallSuper` meta tag. The subclasses will have to call super; otherwise,
+analyzer will show linting error and global key duplication detection might throw unexpected error.
+
+## Migration guide
+
+For example, an app has a `Element` subclass override `forgetChild`
```

Comment:
=> In the following example, an app's `Element` subclass overrode the `forgetChild` method.